src/handwritten/stream_readers.py

```diff
@@ -1833,7 +1867,16 @@ def read_one_sample_multi_line(self, data, timeout=10):
         """
         self._verify_array_digital_lines(data, False, True)
 
-        _read_digital_lines(self._handle, data, 1, timeout)
+        _, samps_per_chan_read, num_bytes_per_samp = self._interpreter.read_digital_lines(
+            self._handle, data, 1, timeout, FillMode.GROUP_BY_CHANNEL.value)
+        
+        ReadDigitalLinesReturnData = (
+        collections.namedtuple(
+            'ReadDigitalLinesReturnData',
+            ['samps_per_chan_read', 'num_bytes_per_samp']))
+
+        ReadDigitalLinesData = ReadDigitalLinesReturnData(
+        samps_per_chan_read, num_bytes_per_samp)    
```

Comment:
None of these namedtuples has any effect. The data is already in the passed-in `data` numpy array and the function doesn't return any values.

Reply:
Removed the named tuples.